python/tvm/tir/parser.py

```diff
@@ -0,0 +1,24 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=unused-import, redefined-builtin
+"""Namespace for Tensor-level IR"""
+
+from . import _ffi_api
+
+
+def astext(code):
```

Comment:
Likely we should not put astext here. Because irnode.astext already redirects to astext in the backend? If we hack relay's astext to redirect. Alternatively overload the astext member function of the TIR Nodes (PrimExpr, PrimFunc).

Reply:
I see. But why tir.Stmt is not a subclass of Node now?

Reply:
It is a subclass of Object in tir/stmt.py

Reply:
there is not special Node class atm. For now let us just add astext to stmt as well.